doc/release-notes.md

```diff
@@ -53,6 +53,25 @@ the Linux kernel, macOS 10.8+, and Windows Vista and later. Windows XP is not su
 Bitcoin Core should also work on most other Unix-like systems but is not
 frequently tested on them.
 
+Notes for 0.15.0
+================
+
+Current SegWit Support
+----------------------
+
+Version 0.15.0 supports adding a segregated witness address via the `addwitnessaddress` RPC, but
+please note that this is a testing/expert RPC, which does not guarantee recovery from backup. Only use
+this RPC if you know what you are doing. More complete support for segregated witness, including
```

Comment:
"including wallet support" sounds like there are other things missing. It's the only thing...